<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.revature.service.ClientServiceTest" tests="5" skipped="0" failures="0" errors="0" timestamp="2022-03-08T23:04:56" hostname="mac.local" time="0.534">
  <properties/>
  <testcase name="test_getClientById_clientDoesNotExist()" classname="com.revature.service.ClientServiceTest" time="0.494"/>
  <testcase name="test_getClientById_invalidId()" classname="com.revature.service.ClientServiceTest" time="0.004"/>
  <testcase name="testGetClientById_positiveTest()" classname="com.revature.service.ClientServiceTest" time="0.015"/>
  <testcase name="testGetAllClients()" classname="com.revature.service.ClientServiceTest" time="0.002"/>
  <testcase name="test_getClientById_sqlExceptionFromDao()" classname="com.revature.service.ClientServiceTest" time="0.009"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
